marxs/source/test/test_farLabPointSource.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from ..labSource import FarLabConstantPointSource as LabSource
+
+
+def test_photon_generation():
+	'''test [FILL IN]
```

Comment:
For test a very short description is enough - nobody will ever import the tests in their program. The comment is only for us that we can quickly see what is tested without reading all the code in the test.